src/java/org/apache/cassandra/hints/HintsWriter.java

```diff
@@ -259,7 +261,7 @@ void append(Hint hint) throws IOException
             if (hintBuffer == buffer)
                 bytesWritten += totalSize;
             else
-                append((ByteBuffer) hintBuffer.flip());
```

Comment:
not necessary

Reply:
Not anymore in Java 11, was needed in Java 8 :)